cli/src/plugin.ts

```diff
@@ -42,15 +44,30 @@ export async function resolvePlugin(name: string): Promise<Plugin | null> {
     const rootPath = resolve('node_modules', name);
     const packagePath = join(rootPath, 'package.json');
     const meta = await readJSON(packagePath);
-    if (!meta || !meta.capacitor) {
+    if (!meta) {
       return null;
+    } else {
```

Comment:
No need for this else, if you remove it it'll make the code cleaner and less idented